docs/users-guide/interactive-dashboards.md

```diff
@@ -0,0 +1,104 @@
+## Interactive dashboards
+
+You can customize what happens when people click on questions in your dashboard.
+
+By default, when you create charts using Metabase's graphical query builder, your charts automatically come with [drill-through capabilities](https://www.metabase.com/blog/drilling-through-data/index.html), which let folks click on a chart to explore further. But if you have a more customized click path in mind, Metabase allows you to customize what happens when a user clicks on a chart or table in your dashboard.
+
+You can set up a dashboard card to:
+
+- Send the user to a custom destination: a dashboard, question, or custom URL.
+- Update a dashboard filter (see [use a chart to filter a dashboard](#Use-a-chart-to-filter-a-dashboard)).
```

Comment:
Seems like either both of these bullets should have anchor links to their respective subsections, or neither of them should.